pywbem/_subscription_manager.py

```diff
@@ -565,17 +590,38 @@ def add_listener_destinations(self, server_id, listener_urls, owned=True,
               :term:`RFC6874`, supporting ``-`` (minus) for the delimiter
               before the zone ID string, as an additional choice to ``%25``.
 
-            Note that the port is required in listener URLs.
+            Note that the port is required in listener URL.
 
             See :class:`~pywbem.WBEMConnection` for examples of valid URLs,
             with the caveat that the port in server URLs is optional.
 
           owned (:class:`py:bool`):
             Defines the ownership type of the created listener destination
-            instances: If `True`, they will be owned. Otherwise, they will be
+            instance: If `True`, it will be owned. Otherwise, it will be
```

Comment:
In the line a above, why is it "ownership type". Isn't it just ownership?

Reply:
NO CHANGE

First, I double checked the docs and source code, and it turns out that so far, we consistently use "ownership type" or "types of ownership", but not just "ownership" (except for two places in the change log). All other occurrences of this are in the `_subscription_manager.py` module.

"Ownership" in the sense of the word only applies to owned instances, but not to permanent instances.
I would like to have a term that is neutral and does not imply just one of the two possible states. Maybe "ownership type" is not optimal for that, but just "ownership" I think is not neutral enough.